crates/fuzz/src/program_test_client_blocking.rs

```diff
@@ -263,6 +269,53 @@ impl FuzzClient for ProgramTestClientBlocking {
             .block_on(self.ctx.banks_client.process_transaction(transaction))?)
     }
 
+    /// Process transaction, returning the result of the tx processing
+    /// in addition to `fn process_transaction()`, this function obtains transaction metadata
+    /// and adds CU units used by the transaction to the `stats_logger`
+    /// # Arguments
+    /// * `transaction` - transaction to process
+    /// * `instruction` - name of the instruction inside the transaction
+    /// * `stats_logger` - statistics logger
+    fn process_transaction_with_metadata(
+        &mut self,
+        transaction: impl Into<VersionedTransaction>,
+        instruction: String,
```

Comment:
Remove the instruction name and the Stats from here, they should be updated in the FuzzTestExecutor macro